coalib/parsing/Globbing.py

```diff
@@ -216,8 +217,8 @@ def fnmatch(name, globs):
     """
     globs = (globs,) if isinstance(globs, str) else tuple(globs)
 
-    if len(globs) == 0:
-        return True
+    if not globs:
```

Comment:
Is this necessary?  It changes the return semantics of this function for edge cases.
best avoided, as it might break coala-quickstart or some other user of these functions which are not being tested by CI.

Reply:
No its not necessary -> I will remove it.
Is there a list, which modules are used by other applications?
That would make it clearer, which functions can be modified safely.